pkg/tools/gen/genkcl_proto.go

```diff
@@ -0,0 +1,92 @@
+package gen
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/emicklei/proto"
+)
+
+var fieldTypeMap = map[string]string{
+	"uint32":              "int",
+	"uint64":              "int",
+	"int32":               "int",
+	"int64":               "int",
+	"sint32":              "int",
+	"sint64":              "int",
+	"string":              "str",
+	"google.protobuf.Any": "any",
+}
+
+// genKclFromProtoData
+func (k *kclGenerator) genKclFromProtoData(w io.Writer, filename string, src interface{}) error {
```

Comment:
> I suggest using the template here instead of using string concatenation https://github.com/kcl-lang/kcl-go/blob/main/pkg/tools/gen/template.go#L50 like the jsonschema to kcl schema and terraform to kcl schema

proto定义时无法赋值，所以我参考的是https://github.com/kcl-lang/kcl-go/blob/00adadd8c6f2e1dbee45642017d16369cff03c6b/pkg/tools/gen/genkcl.go#L95方法